.gitignore

```diff
@@ -19,6 +19,9 @@ zeppelin-server/derby.log
 
 lens/lens-cli-hist.log
 
+# bin file
+bin/derby.log
```

Comment:
we have `spark/derby.log` above, do we know why we need one for `bin/derby.log`?